docs/scripting/callbacks/OnRconCommand.md

```diff
@@ -39,17 +39,19 @@ public OnRconCommand(cmd[])
 
 ## Notes
 
-:::tip
+:::warning
+
+The /rcon prefix is not included in the cmd parameter when a player types a command. If you use the print function here, it will send a message to both the player who typed the command in-game and the server log.
 
-"/rcon " is not included in "cmd" when a player types a command. If you use the "print" function here, it will send a message to the player who typed the command in-game as well as the log. This callback is not called when the player is not logged in as RCON admin. When the player is not logged in as RCON admin and uses /rcon login, this callback will not be called and OnRconLoginAttempt is called instead. However, when the player is logged in as RCON admin, the use of this command will call this callback.
+This callback is not called if the player is not logged in as an RCON admin. When a player uses /rcon login to log in, this callback will not be called, instead, OnRconLoginAttempt is called. Once logged in as an RCON admin, any subsequent commands will trigger this callback.
 
 :::
 
-:::warning
+:::tip
```

Comment:
I believe this is a 'warning' instead of a 'tip'.

Reply:
I agree. Done